### ISSUE TITLE
proxy.sh exists in the docker hub documentation, but not in the image

### ISSUE THREAD

**Issue by Ueland** [haugene/docker-transmission-openvpn#501]
Upon running the image with Proxy.sh, which is stated supported on the hub-page, i am greeted with:

> Could not find OpenVPN provider: PROXYSH
> Please check your settings.

**Comment by Ueland** [comment]
For those noticing the same issue, as of now, proxy.sh support is only in the dev-tag :+1:

**Comment by haugene** [comment]
Yeah. Docker hub uses the README from the last build. So when dev is built the description changes even though the feature might not be on the "latest" tag yet.
